packages/identifier-resolution/src/agent/IdentifierResolution.ts

```diff
@@ -145,7 +147,12 @@ export class IdentifierResolution implements IAgentPlugin {
   ): Promise<ExternalIdentifierCoseKeyResult> {
     return (await this.identifierResolveExternal({ ...args, method: 'cose_key' }, context)) as ExternalIdentifierCoseKeyResult
   }
+  
   private async identifierExternalResolveByJwk(args: ExternalIdentifierJwkOpts, context: IAgentContext<any>): Promise<ExternalIdentifierJwkResult> {
     return (await this.identifierResolveExternal({ ...args, method: 'jwk' }, context)) as ExternalIdentifierJwkResult
   }
+  
+  private async identifierExternalResolveByOIDFEntityId(args: ExternalIdentifierOIDFEntityIdOpts, context: IAgentContext<IOIDFClient>): Promise<ExternalIdentifierOIDFEntityIdResult> {
```

Comment:
i am not sure about the structure used IAgentContext<IOIDFClient>, if you look at others it is mIAgentContext<any> and later on we check if the context has a certain plugin see:

(!contextHasPlugin<IResolver & IDIDManager>(context, 'resolveDid')) {